tests/denops/testdata/no_check/cli_error_on_issue_401.ts

```diff
@@ -0,0 +1 @@
+import * as _ from "jsr:@std/async@1.0.9999";
```

Comment:
The version 1.0.9999 appears to be **correct**, so:

1. Add a comment explaining that the version is **incorrect**.
2. Use a version like 0.0.0, 1.0.0-invalid, or any other format that clearly indicates it is **incorrect**.

Reply:
Changed clarify.